proceeding/text/idea.tex

```diff
@@ -18,3 +18,20 @@ \section{Idea}
 \item Open source, open data - allows for 1. users to download any data from our website, and 2. for other developers to contribute to the code.
 
 \end{itemize}
+
+
+\paragraph{}    gamma-sky.net is a one-stop resource for perusing images and catalogs but also closely examining a specific gamma-ray source.
+    Although it was mainly built for the astronomical community, the webpage also targets the general public through a
+    user-friendly interface and a clean information layout, all of which is compiled under cutting-edge web tools. \par
+
+\paragraph{}    Individuals who access the website via any modern internet browser will be welcomed with the Map View page.
+    We overlaid multi-wavelength survey images, most of which are all-sky images, onto a three-dimensional sphere.
+    The map features pan-and-zoom functionality for easily navigating and quickly browsing the sky. Gamma-ray sources from
+    our catalog data have been pinpointed onto the sphere, as shown in Figure 1. This setup allows the user to understand
+    the visual context of sources in relation to one another. It also enables the user to easily contrast sources across catalogs.
+    gamma-sky.net additionally embodies a Catalog View, which incorporates more detailed information for each of the sources in our catalogs.
```

Comment:
There should be some reference to a Figure with the catalog view, when you describe the catalog view in the text, no?